assets/js/blocks/mini-cart/edit.tsx

```diff
@@ -212,15 +286,45 @@ const Edit = ( { attributes, setAttributes }: Props ): ReactElement => {
 					</BaseControl>
 				</PanelBody>
 			</InspectorControls>
+			{ colorGradientSettings.hasColorsOrGradients && (
+				// @ts-to-do: Fix outdated InspectorControls type definitions in DefinitelyTyped and/or Gutenberg.
+				<InspectorControls group="color">
+					{ colorSettings.map(
+						( { onChange, label, value, resetAllFilter } ) => (
+							<ColorGradientSettingsDropdown
+								key={ `mini-cart-color-${ label }` }
+								__experimentalIsRenderedInSidebar
+								settings={ [
+									{
+										colorValue: value,
+										label,
+										onColorChange: onChange,
+										isShownByDefault: true,
+										resetAllFilter,
+										enableAlpha: true,
+									},
+								] }
+								panelId={ clientId }
+								{ ...colorGradientSettings }
+							/>
+						)
+					) }
+				</InspectorControls>
+			) }
 			<Noninteractive>
 				<button className="wc-block-mini-cart__button">
 					{ ! hasHiddenPrice && (
-						<span className="wc-block-mini-cart__amount">
+						<span
+							className="wc-block-mini-cart__amount"
+							style={ { color: priceColorValue } }
```

Comment:
One downside of doing this _always_ for the colors is it means this can never be set via presets from a theme (or utilize Global Styles). It also means that style sheets can't really override this either because this is always an inline style color attribute. 

There's an [example here](https://github.com/x3p0-dev/x3p0-progress/blob/1.0/resources/js/color/panel-color.js) of a custom color panel that looks like it accounts for the above and might be a goo choice for implementation here. However, I'm also currently working on implementing a custom color panel for the Price block and it might be something worth abstracting for re-using across various blocks we might want custom colors (price, ratings, mini-cart etc).

Reply:
Thanks for sharing Justin's solution. I agree that making this compatible with theme presets or Global Styles is a worthwhile improvement (and standardizing it across our codebase would be ideal).

I'll open an issue for this. Please do share any code you decide to use for the Price block and we can model the solution for the Mini Cart based on that.